File: Python Word Examples/ShapesApiDemo/getShapeFormat.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.ShapesApi(configuration)

file_name = 'Shape.docx'
paragraph_path = 'Section/0/Body/0/Paragraph/0'
index = 12
result = api.get_shape_format(file_name, paragraph_path, index, folder="input")

File: JavaScript Word Examples/ShapesApiDemo/getShapeFormat.js
(function (){
    var appId = "your id";
    var appKey = "your key";   
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ShapesApi(apiClient);

    var name = "Shape.docx";
    var paragraphPath = "Section/0/Body/0/Paragraph/0";
    var index = 12;
    var opts = { folder: 'input' };
    instance.getShapeFormat(name, paragraphPath, index, opts,
        function (error, data, response) {
            if (error) throw error;
        });
})();